client/app/locales/locale-de.json

```diff
@@ -0,0 +1,10 @@
+{
+  "LOGIN": {
+    "TITLE": "Login",
+    "EMAIL": "E-Mail",
+    "PASSWORD": "Passwort",
+    "WRONG": "Die Anmeldedaten sind fehlerhaft. Bitte überprüfe sie sorgfältig.",
+    "SUBMIT": "Login",
+    "SIGNUP": "Neuanmeldung"
+  }
+}
```

Comment:
I used angular-translate at letsmake, and I didn't do very well at keeping the translations keys in order, and it got confusing after a while to work out where a particular piece of text would appear on the website (if at all).

I don't have a clever solution, but just to keep it in mind!

Maybe if there is something that can parse the js (even just a regexp) and extract the keys that are used, could be useful. Dunno. Might just be something we have to keep on top of semi-manually.